internal/persistence/entgo/postgres.go

```diff
@@ -42,8 +42,12 @@ func init() {
 	persistence.RegisterStorage("postgres", &PostgreSQL{})
 }
 
-// example dsn: "user=<username> password=<password> host=localhost port=5432 database=wfx sslmode=disable"
-// see https://github.com/jackc/pgx/blob/master/stdlib/sql.go
+// Initialize sets up the PostgreSQL database connection using the provided DSN (options), runs migrations, and initializes the ent client.
```

Comment:
That should read [...] en**d** client, right?

Reply:
No, the library is called `entgo` and the client is really called `ent.Client` :)